runners/gradle-plugin/build.gradle.kts

```diff
@@ -88,9 +88,9 @@ tasks.withType<PublishToMavenRepository>().configureEach {
 }
 
 afterEvaluate { // Workaround for an interesting design choice https://github.com/gradle/gradle/blob/c4f935f77377f1783f70ec05381c8182b3ade3ea/subprojects/plugin-development/src/main/java/org/gradle/plugin/devel/plugins/MavenPluginPublishPlugin.java#L49
-    configureSpacePublicationIfNecessary("pluginMaven", "dokkaGradlePluginPluginMarkerMaven")
-    configureSonatypePublicationIfNecessary("pluginMaven", "dokkaGradlePluginPluginMarkerMaven")
-    createDokkaPublishTaskIfNecessary()
+//    configureSpacePublicationIfNecessary("pluginMaven", "dokkaGradlePluginPluginMarkerMaven")
+//    configureSonatypePublicationIfNecessary("pluginMaven", "dokkaGradlePluginPluginMarkerMaven")
+//    createDokkaPublishTaskIfNecessary()
 }
```

Comment:
I'm not quite sure if the changes I've made are equivalent to this existing config, and it's difficult to check without actually publishing.